src/inspector_profiler.cc

```diff
@@ -265,6 +265,44 @@ void V8CpuProfilerConnection::End() {
   DispatchMessage("Profiler.stop");
 }
 
+std::string V8HeapProfilerConnection::GetDirectory() const {
+  return env()->heap_prof_dir();
+}
+
+std::string V8HeapProfilerConnection::GetFilename() const {
+  return env()->heap_prof_name();
+}
+
+MaybeLocal<Object> V8HeapProfilerConnection::GetProfile(Local<Object> result) {
```

Comment:
@ofrobots See here and `V8ProfilerConnection::GetProfile()`. Because the response returned is:

```
{
  "result": {
    "profile": {
      // ... what we need
    }
}
```

We need to do `JSON::Parse` and then `JSON::Stringify` to get to the profile. It would be better if the profile is formatted directly from the C++ data and since we already know about the schema of the returned object, we can serialize it more efficiently (similar to how heap snapshots are serialized in the API).